styleguide/_themes/derek/scss/components/_ceiling.scss

```diff
@@ -44,13 +46,11 @@ $ceiling-font-size: 11px;
     margin-bottom: 0;
 
     > li{
-      margin: $ceiling-margin 0;
-      padding: 0;
-      &:after {
-        content: "|";
-      }
-      &:last-child::after {
-        content: " ";
+      margin: floor($ceiling-margin + 2) 0;
```

Comment:
Fair enough. It's probably a non-issue honestly since it doesn't affect the final CSS one way or another.